WorkloadManagementSystem/Client/PilotAgentsClient.py

```diff
@@ -0,0 +1,40 @@
+""" Class that contains client access to the PilotAgents handler. """
+########################################################################
+# $Id$
+# $HeadURL$
+########################################################################
+__RCSID__ = "$Id$"
+
+from DIRAC.Core.Base.Client                         import Client
+
+class PilotAgentsClient( Client ):
+  
+  def __init__( self, **kwargs ):
+    Client.__init__( self, **kwargs )
+    self.setServer( 'WorkloadManagement/PilotAgents' )
+    self.pilotAgenstHandler = self._getRPC()
+    
+  def addPilotsLogging( self, pilotUUID, status, minorStatus, timeStamp, source, pilotID ):
+    
```

Comment:
There's a need for a docstring with documentation here.

If pilotID is present, it means the source is different from "pilot", because pilots don't know about pilotID, but only know about their pilotUUID.